content/blog/2020-07-22-july-20-community-gems.md

```diff
@@ -0,0 +1,183 @@
+---
+title: July '20 Community Gems
+date: 2020-07-31
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  getting started with CML, configuring your DVC cache, and how to request a 
+  tutorial video.
+picture: 2020-07-31/Gems_July_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/july-20-community-gems/460
+tags:
+  - CML
+  - Bitbucket
+  - GCP
+  - global remote
+  - cache
+---
+
+Here are some of our top Q&A's from around the community. With the launch of
+[CML](https://cml.dev) earlier in the month, we've got some new ground to cover!
+
+## DVC questions
+
+### [Q: Recently, I set up a global DVC remote. Where can I find the config file?](https://discordapp.com/channels/485586884165107732/563406153334128681/717673618217238598)
+
+When you
+[create a global DVC remote](https://dvc.org/doc/command-reference/remote/list#options),
+a config file will be created in `~/.config/dvc/config` instead of in your
```

Comment:
`instead of in` -  something is missing here?

also, we should probably mention location on Windows

Reply:
good idea but i can't find in the docs anything about the cache location in windows- can you point me to it, @shcheklein?

Reply:
you mean config ... good point ... @efiop do you remember this by chance?

Reply:
```
ruslan  4:10 PM
IIRC
C:\Users\<username>\AppData\Local\iterative\dvc\config
C:\ProgramData\iterative\dvc\config
```

Reply:
Does this mean the config is saved in two places on a Windows system, or one is an an alias of the other? Or it will be in one of those two locations?

Reply:
`C:\Users\<username>\AppData\Local\iterative\dvc\config` corresponds to `global` I guess.